prow/github/client.go

```diff
@@ -3561,6 +3562,70 @@ func (c *client) CreateFork(owner, repo string) (string, error) {
 	return resp.Name, err
 }
 
+// EnsureFork checks to see that there is a fork of org/repo in the forkedUsers repositories.
+// If there is not, it makes one, and waits for the fork to be created before returning.
+func (c *client) EnsureFork(forkingUser, org, repo string) (string, error) {
```

Comment:
Nit: Could you godoc that the returned string is the repos name and that it might be different from the original name due to a naming conflict?